### DISCUSSION TITLE
What is the upload size?

### DISCUSSION THREAD

**Discussion by resherry2020** [requarks/wiki#7117] [Help / Questions] [answered]
I'm curious about the storage limit available on Wiki.js. I'd like to know if it can support my future usage needs. I see in System Info that the Total RAM is 1.95 GB. Does this mean I can utilize up to 1.95 GB? Is this determined by the size of my server?

**Comment by NGPixel** [comment, marked as answer]
No, RAM has nothing to do with your wiki.js size.

Disk space is what matters.

**Reply by resherry2020** [reply]
Thanks, How could I know my disk space size?
